### PR TITLE
Redesign catalog page

### PR DESCRIPTION
**Pull request by nihco2**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Catalog page redesign |
| Type? | improvement |
| Category? | BO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-1130 |
| How to test? | Check directly with Léa or compare with mockups |

### CODE REVIEW

**Comment by vincentbz** [comment]
Tested and ok, thanks !

File: src/PrestaShopBundle/Resources/views/Admin/Product/catalog.html.twig [hunk 1]
```diff
@@ -171,7 +252,7 @@
                       } %}
                     </th>
                   {% endif %}
-                  <th></th>
+                  <th style="width: 6%"></th>
```
Comment (Quetzacoalt91) [inline src/PrestaShopBundle/Resources/views/Admin/Product/catalog.html.twig]: You should avoid this kind of style definition which cannot be overwritten

File: src/PrestaShopBundle/Resources/views/Admin/Product/catalog.html.twig [hunk 2]
```diff
@@ -243,18 +324,22 @@
...
-                      {% endif %}
+                        <input type="button" class="btn btn-tertiary-outline" name="products_filter_position_asc" value="{{ "Reorder"|trans({}, 'Admin.Actions') }}" onclick="productOrderPrioritiesTable();" />
+                        {% else %}
+                        <input type="button" id="bulk_edition_save_keep" class="btn" onclick="bulkProductAction(this, 'edition');" value="{{ "Save & refresh"|trans({}, 'Admin.Actions')|raw }}" />
+                    {% endif %}
+
                     </th>
                   {% endif %}
-                  <th>
+                  <th style="width: 12%">
```
Comment (Quetzacoalt91) [inline src/PrestaShopBundle/Resources/views/Admin/Product/catalog.html.twig]: Same here, avoid style inline

File: admin-dev/themes/new-theme/template/page_header_toolbar.tpl [hunk 3]
```diff
@@ -35,15 +35,31 @@
...
+            data-toggle="tooltip"
+            data-placement="bottom"{/if}
+          >
             <i class="material-icons">{$btn.icon}</i>
             <span class="title">{$btn.desc|escape}</span>
           </a>
         {/if}
       {/foreach}
       {if isset($toolbar_btn['modules-list'])}
         {* TODO: REFACTOR ALL THIS THINGS *}
```
Comment (mickaelandrieu) [inline admin-dev/themes/new-theme/template/page_header_toolbar.tpl]: LOL !

File: src/PrestaShopBundle/Resources/views/Admin/Product/catalog.html.twig [hunk 4]
```diff
@@ -40,50 +72,102 @@
...
       <div class="row">
-        <div id="product_catalog_category_tree_filter" class="col-md-8">
-          {{ form_widget(categories) }}
+        <div class="col-md-1">
+          <div class="checkbox">
+            <label>
+              <input
+                type="checkbox"
+                id="bulk_action_select_all"
+                onclick="$('#product_catalog_list').find('table td.checkbox-column input:checkbox').prop('checked', $(this).prop('checked')); updateBulkMenu();"
```
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Resources/views/Admin/Product/catalog.html.twig]: Would you mind to store this behavior inside this own function ? Maybe ``syncCheckboxes()`` or something ?

File: src/PrestaShopBundle/Resources/views/Admin/Product/catalog.html.twig (hunk 1 above)
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Resources/views/Admin/Product/catalog.html.twig]: uh_uh

File: admin-dev/themes/new-theme/template/page_header_toolbar.tpl (hunk 3 above)
Comment (Quetzacoalt91) [inline admin-dev/themes/new-theme/template/page_header_toolbar.tpl]: We should have an URL instead:

REFACTOR ALL THE THINGS

**Comment by Shudrum** [comment]
Cherry picket on #6013, thank you @nihco2!

**Pull request closed without merging**